packages/backend/src/utils/constants.ts

```diff
@@ -3,7 +3,8 @@ import { join } from "path";
 import { homedir } from "os";
 
 class ConstantsUtil {
-  public IS_IN_BAS = !isEmpty(get(process, "env.WS_BASE_URL"));
+  public IS_IN_BAS =
+    !isEmpty(get(process, "env.WS_BASE_URL")) || get(process, "env.TENANT_PLAN", "false") === "personal-edition";
```

Comment:
It's OSS, but here you expose sensitive data. consider relying on a different condition.